### PR TITLE
Timber Wolf - updated via keymap, reduced eeprom writing from backlight code

### PR DESCRIPTION
**Pull request by Croktopus**
## Description

- changed the via.c keymap to be more useful as a stock keymap and require less adjustment
- made a few other small adjustments to match up with via recommended settings
- modified custom caps lock code to eat less eeprom

sorry for spamming PRs, keyboards are shipping so I want all the firmware to be as good as possible before the first package arrives.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/metamechs/timberwolf/rules.mk [hunk 1]
```diff
@@ -31,5 +31,6 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
 ENCODER_ENABLE = yes        # Enable encoder support
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/rules.mk:34]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/metamechs/timberwolf/rules.mk [hunk 2]
```diff
@@ -31,5 +31,6 @@ AUDIO_ENABLE = no           # Audio output on port C6
 FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
 HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/rules.mk:32]:
```suggestion
```

File: keyboards/metamechs/timberwolf/timberwolf.c [hunk 3]
```diff
@@ -16,12 +16,15 @@
 
 #include "timberwolf.h"
 
+
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/timberwolf.c:19]:
```suggestion
```

File: keyboards/metamechs/timberwolf/timberwolf.c [hunk 4]
```diff
@@ -16,12 +16,15 @@
...
-      backlight_enable();
-    } else {
-		backlight_disable();
+	if(runDefault) {
+		if (led_state.caps_lock) {
+		  backlight_level_noeeprom(get_backlight_level());
+		} else {
+			backlight_set(0);
+		}
 	}
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/timberwolf.c:28]:
```suggestion
    if(runDefault) {
        if (led_state.caps_lock) {
            backlight_level_noeeprom(get_backlight_level());
        } else {
            backlight_set(0);
        }
    }
```

File: keyboards/metamechs/timberwolf/rules.mk [hunk 5]
```diff
@@ -28,8 +28,7 @@ RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 MIDI_ENABLE = no            # MIDI support
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/rules.mk:30]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Comment by fauxpark** [comment]
Thanks!

**Comment by Croktopus** [comment]
thankee!

**Pull request merged**